### ISSUE TITLE
Full-screen with HTML5 videos don't works as expected

### ISSUE THREAD

**Issue by clopez** [piroor/treestyletab#457]
Hello,

Running Firefox 18 on Debian Wheezy with Tree Style Tab extension.

When I click on a HTML5 video to see it fullscreen (for example here: http://www.quirksmode.org/html5/tests/video.html ) the full screen mode don't works as expected.

The full screen video don't goes over the vertical panel of tree style tab. Check:

http://ftp.neutrino.es/treestyletab.jpg

If I disable tree style tab then the html5 full screen video goes over the default tabs.

**Comment by AlexCzar** [comment]
This is a problem for me, I do want TTT to be visible when Firefox is full-screen, but not when video is full-screen, these are two different use cases. Please reopen.
Also, when video becomes full-screen TTT is not even shown properly, the tabs themselves disappear and the TTT bar becomes just a blank space.

**Comment by masterpi314** [comment]
I started having this after upgrade to 41.0.1, and also even more annoyingly, tab bar was auto-hiding after returning from fullscreen.  Maybe Firefox switched the flag they're sending for a fullscreen event, or it's just a toggle event sent and the plugin isn't tracking state right.  I worked around it by inverting my settings so that it's set to auto-hide in "window mode" and always show in "fullscreen".
